addon/utils/css-transitions/has-transitions.js

```diff
@@ -0,0 +1,5 @@
+export default function hasTransitions(element) {
+	const { transitionProperty, transitionDuration } = window.getComputedStyle(element);
```

Comment:
Link to [CanIUse](http://caniuse.com/#search=getComputedStyle). It seems to work on modern browsers.